template/{{.input_root_dir}}/{{template `project_name_alphanumeric_underscore` .}}/README.md.tmpl

```diff
@@ -37,7 +37,7 @@ contained in the following files:
 │   │
 │   ├── databricks.yml          <- databricks.yml is the root bundle file for the ML project that can be loaded by databricks CLI bundles. It defines the bundle name, workspace URL and resource config component to be included.
 │   │
-{{ if and (eq .input_include_feature_store `no`) (eq .input_include_mlflow_recipes `no`) -}}
+{{- if and (eq .input_include_feature_store `no`) (eq .input_include_mlflow_recipes `no`) -}}
```

Comment:
Could you try generating the file?
I suspect it will generate something like:
```
│ ││ ├── training <- Training folder contains Notebook
```

Reply:
The original state is 
```
{{ if .. -}}
...
{{ else if .. -}}
...
{{ else -}}
...
{{ end -}}
```
Every condition is on the separate line(same as current state).

Reply:
Yup great call out, removed the following `-` to ensure the space doesn't get deleted so it looks better now 
<img width="1326" alt="Screenshot 2024-02-07 at 12 41 13 PM" src="https://github.com/databricks/mlops-stacks/assets/87999496/7f907c4e-769f-4009-9235-adc9ee8550c7">